### ISSUE TITLE
Backslashes in paths causing swagger parsing error

### ISSUE THREAD

**Issue by tamethecomplex** [thiagobustamante/typescript-rest-swagger#3]
Hi, thanks for putting this project out there, I think it's exactly what I've been looking for.  I am trying to use this tool in conjunction with https://www.npmjs.com/package/typescript-rest and  https://www.npmjs.com/package/swagger, and the backslashes in the generated paths are causing the YAML not to parse in the swagger editor.  Just wondering why the paths are generated with backslashes rather than forward slashes?  The example shown on the swagger spec page shows a forward slash: http://swagger.io/specification/#pathsObject

Generated using swaggerGen:
```
paths:
    '\hello\{name}':
```

**Comment by tamethecomplex** [comment]
I see now this is caused by the use of pathUtil.join at this line of generator.ts:

`let path = pathUtil.join('/', (controller.path ? controller.path : ''), method.path);`

And that per the node path documentation here: https://nodejs.org/api/path.html#path_path_join_paths, on Windows the path segment separator is '\' resulting in this behavior since I'm on Windows.

I patched this issue locally in generator.ts by adding this line to replace all occurrences of '\' with '/':

```
let path = pathUtil.join('/', (controller.path ? controller.path : ''), method.path);
path = path.replace(/\\/g,'/')  // <---------- Added line ------------
paths[path] = paths[path] || {};
```

I did a quick search and didn't find a way to change the behavior of path to use POSIX separators instead of Windows separators when running on Windows.  So maybe the above is the easiest way to solve this for folks running your utility on Windows?

**Comment by thiagobustamante** [comment]
Hi, 

Thanks for your feedback! I will take a look at this problem.

**Comment by thiagobustamante** [comment]
I commited this fix and published 0.0.7 version. Could you please try this new version?

**Comment by tamethecomplex** [comment]
Works perfectly now in version 0.0.7.  Thanks!
